ownCloud/Client/ClientRootViewController.swift

```diff
@@ -155,32 +166,28 @@ class ClientRootViewController: UITabBarController {
 
 		self.tabBar.applyThemeCollection(Theme.shared.activeCollection)
 
-		self.viewControllers = [filesNavigationController] as? [UIViewController]
-	}
-
-	func logoutBarButtonItem() -> UIBarButtonItem {
-		let barButton = UIBarButtonItem(title: "Disconnect".localized, style: .plain, target: self, action: #selector(logout(_:)))
-		barButton.accessibilityIdentifier = "disconnect-button"
-		return barButton
-	}
-
-	@objc func logout(_: Any) {
-		self.closeClient()
+        self.viewControllers = [filesNavigationController] as? [UIViewController]
```

Comment:
@hosy is it needed as `UINavigationController` inherits from `UIViewController`?

Reply:
Yes, because there is a compiler error, if I remove `as? [UIViewController]`

Reply:
@hosy ok, fine